scripts/spread-hunter-menu.ps1

```
@@ -1027,6 +1031,189 @@ function Show-CheckoutIdentity {
+    needed and to prove the environment is clean before starting. #>
+    if (Test-LivePort) { return $true }
+    if ($null -ne (Get-DashInstance)) { return $true }
+    if ($null -ne (Get-ShadowDashInstance)) { return $true }
+    if (Test-Path $ProcsFile) {
+        try {
+            $saved = Get-Content $ProcsFile -Raw | ConvertFrom-Json
+            foreach ($name in @("filter", "query", "decide")) {
+                $info = Get-ServiceEntry -Saved $saved -Key $name
+                if ($info -and $info.pid -and (Test-PidAlive -ProcessId $info.pid)) { return $true }
+            }
+        } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)